### DISCUSSION TITLE
copilot login to github error

### DISCUSSION THREAD

**Discussion by luoxiaofeng1029** [community/community#40896] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

![image](https://user-images.githubusercontent.com/50006313/205850589-b95d453a-9220-48cd-be43-58a36aa10b2d.png)
![image](https://user-images.githubusercontent.com/50006313/205850624-f6ef3335-8335-4957-a970-80e26baf6b04.png)
please help me

**Comment by Akash1134** [comment, marked as answer]
Hi @luoxiaofeng1029 , Is it possible that your employer, internet service provider, or government, may be intercepting your SSL traffic or you are using a proxy server to connect?

**Reply by AthulMuralidhar** [reply]
i have the same error and im not behind any proxy :/

**Reply by luoxiaofeng1029** [reply]
今天好了,昨天特殊

**Comment by DiamondMofeng** [comment]
今天github被列入诈骗网站了。。dns被污染了，换个dns试试

**Reply by luoxiaofeng1029** [reply]
是的,今天好了

**Comment by iamcjwilliams** [comment]
@luoxiaofeng1029  Could you provide a few more details on how this can be replicated?

**Reply by luoxiaofeng1029** [reply]
今天好了,昨天应该是特殊日子,被列入诈骗站点了

**Reply by butterl** [reply]
@iamcjwilliams  @Akash1134 
I ran into same kind of issue，but it triggered by my Clion + copliot upgrade together,  it not seems to be a  DNS issue

since the one have this self-signed  cert issue        Clion version  #CL-223.7571.17 + copilot 1.1.37.2188
and the one without this issue(same machine )      Pycharm #PY-222.4345.23 + 1.1.33.1980

error code is a little bit different :

```
Request signInInitiate failed with message: self signed certificate in certificate chain, request id: 7, error code: -32603
```

**Comment by siddharthgoel88** [comment]
Checkout https://sidd.io/2023/01/github-copilot-self-signed-cert-issue/
